SQL docs review (#389)
hazelcast/hz-docs: docs/modules/sql/pages/querying-maps-sql.adoc

sql.execute("SELECT * FROM employees WHERE employee.status = ?", EmployeeStatus.VACATION);
```

== Streaming Map Changes

> At the moment, you cannot run queries streaming the changes on maps with SQL.

Reviewer: ```suggestion
At the moment, you cannot use SQL to run queries that stream changes from maps.
```
